src/cpu_tables.c

```diff
@@ -523,6 +523,10 @@ CPU cpus_Slot1_100MHz[] =
         {"Pentium II/350", CPU_PENTIUM_2, fpus_builtin, 20, 350000000, 3, 33333333, 0x651, 0x651, 0, CPU_SUPPORTS_DYNAREC | CPU_REQUIRES_DYNAREC, 18,18,9,9, 42},
         {"Pentium II/400", CPU_PENTIUM_2, fpus_builtin, 20, 400000000, 3, 33333333, 0x652, 0x652, 0, CPU_SUPPORTS_DYNAREC | CPU_REQUIRES_DYNAREC, 18,18,9,9, 48},
         {"Pentium II/450", CPU_PENTIUM_2, fpus_builtin, 20, 450000000, 3, 33333333, 0x652, 0x652, 0, CPU_SUPPORTS_DYNAREC | CPU_REQUIRES_DYNAREC, 18,18,9,9, 54},
+        {"Pentium II/550", CPU_PENTIUM_2, fpus_builtin, 20, 550000000, 3, 33333333, 0x652, 0x652, 0, CPU_SUPPORTS_DYNAREC | CPU_REQUIRES_DYNAREC, 18,18,9,9, 70},
+        {"Pentium II/700", CPU_PENTIUM_2, fpus_builtin, 20, 700000000, 3, 33333333, 0x652, 0x652, 0, CPU_SUPPORTS_DYNAREC | CPU_REQUIRES_DYNAREC, 18,18,9,9, 88},
+        {"Pentium II/800", CPU_PENTIUM_2, fpus_builtin, 20, 800000000, 3, 33333333, 0x652, 0x652, 0, CPU_SUPPORTS_DYNAREC | CPU_REQUIRES_DYNAREC, 18,18,9,9, 100},

```

Comment:
What about some 1.5GHz one too, for cases when that's enough and one wants to avoid maximal slowdown?

Reply:
Makes sense, I just PRed the ones that I added while experimenting last year.

Is it a bad idea to have too many CPU options? Do we want to decide which ones to go with?